tools/install_ssl_configuration.sh

```diff
@@ -0,0 +1,320 @@
+SCRIPT_PATH="${BASH_SOURCE[0]}"
```

Comment:
note on variable naming: you should avoid all-caps variable names, see e.g. https://stackoverflow.com/a/673940 (and related links there)